Shoko.Server/Commands/AniDB/CommandRequest_GetFile.cs

```diff
@@ -138,34 +138,36 @@ protected override void Process()
     public void CreateLanguages(ResponseGetFile response)
     {
         using var session = DatabaseFactory.SessionFactory.OpenSession();
-        if ((response?.AudioLanguages?.Count ?? 0) >
-            0) //Only create relations if the origin of the data if from Raw (WebService/AniDB)
+        using var trans = session.BeginTransaction();
+        // Only update languages if we got a response
+        if (response?.AudioLanguages is not null)
```

Comment:
This will remove all existing languages even if the response doesn't contain any (new) languages (to replace the list with).

If this is the intended behavior then this change is fine, otherwise it would be wise to revert this line.